packages/accounts-base/accounts_server.js

```diff
@@ -525,11 +533,15 @@ Ap._initServerMethods = function () {
   };
 
   methods.logout = function () {
+    var user = accounts.users.findOne(this.userId)
```

Comment:
Can we only fetch the user if there is a logout hook? Basically put this inside the `if`